en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,77 @@
-# PX4 Continuous Integration
+# PX4 Continuous Integration / Automation
 
-PX4 builds and testing are performed using GitHub actions and a Jenkins server.
\ No newline at end of file
+PX4 has multiple automated CI build that runs to keep different systems in sync. This document describes how they are configured & connected, to hopefully clear-up confusion in how everything is connected!
+
+:::note
+CI scripts are subject to change, and the links below may get broken in the future. If that happens, please contribute the fix for the documentation!
+:::
+
+## PX4-Autopilot Pull Request CI tests
+For every [Pull-Request](https://github.com/PX4/PX4-Autopilot/pulls), there are automated CI that checks the PR.
+Most of the workflows are done with Github workflows, which can be found [here](https://github.com/PX4/PX4-Autopilot/tree/main/.github/workflows).
+
+- `compile_nuttx.yml`: Builds NuttX flight controller board targets. It can detect Flash overflows and code syntax errors
+- `sitl_tests.yml`: Executes [MAVSDK SITL Tests](./integration_testing_mavsdk.md). It can detect functional bugs in the code (e.g. VTOL doesn't return to home)
+- `mavros_mission_tests.yml`: Executes Mission plans using MAVROS. It can check for functional compatibility between MAVROS and PX4
+- `checks.yml`: Runs various make targets (using `make` command). The commands for each targets are documented in the [Makefile](https://github.com/PX4/PX4-Autopilot/blob/main/Makefile)
+  - `check_format`: Checks the coding style format using Clang Tidy ?
+  - `airframe_metadata`: Builds airframe XML metadata
+  - `parameters_metadata`: Builds parameter XML metadata
+  - `module_documentation`: Builds module documentation
+  - `tests`: Runs unit tests
+  - `tests_coverage`: Runs coverage tests
+  - ... (other auxiliary tests)
+- `ekf_functional_change_indicator.yml`: Check if there's any functionality change in EKF ?
+- `ekf_update_change_indicator.yml`: Check if there's any functionality change in EKF ?
```

Comment:
I wasn't sure what this really does. @bresch do you have comment on this?

Reply:
The change indicator feeds the data of a log through the estimator and compares the output with the output that we got before the changes (saved in a csv file). It throws an error if the result is numerically different (with some tolerance). This is expected to fail for a functional change but is useful for some pure refactoring work.

Reply:
Thanks!